imap/imapd.c

```diff
@@ -6407,6 +6407,20 @@ static void cmd_search(const char *tag, const char *cmd)
             searchargs->returnopts = SEARCH_RETURN_ALL;
         }
 
+        struct conversations_state *cstate = NULL;
+        if (searchargs->fuzzy_depth &&
+            config_getswitch(IMAPOPT_CONVERSATIONS) &&
+            config_getenum(IMAPOPT_SEARCH_ENGINE) == IMAP_ENUM_SEARCH_ENGINE_XAPIAN) {
+            /* Open the cstate here to avoid doing it for every mailbox
+               and avoid lock conflicts */
+            int r = conversations_open_user(imapd_userid, 1, &cstate);
```

Comment:
would be good to comment `/*readonly*/` or `/*shared*/` on the second arg.

Reply:
Sure.